src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -161,24 +142,20 @@ const getQuickActionTitle = (action: QuickActionName): TranslationPaths => {
  * Responsible for rendering the {@link PopoverMenu}, and the accompanying
  * FAB that can open or close the menu.
  */
-function FloatingActionButtonAndPopover(
-    {
-        onHideCreateMenu,
-        onShowCreateMenu,
-        isLoading = false,
-        allPolicies,
-        quickAction,
-        quickActionReport,
-        quickActionPolicy,
-        session,
-        personalDetails,
-        hasSeenTrackTraining,
-    }: FloatingActionButtonAndPopoverProps,
-    ref: ForwardedRef<FloatingActionButtonAndPopoverRef>,
-) {
+function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu}: FloatingActionButtonAndPopoverProps, ref: ForwardedRef<FloatingActionButtonAndPopoverRef>) {
     const styles = useThemeStyles();
+    const theme = useTheme();
     const {translate} = useLocalize();
+    const [isLoading] = useOnyx(ONYXKEYS.IS_LOADING_APP);
```

Comment:
```suggestion
    const [isLoading = false] = useOnyx(ONYXKEYS.IS_LOADING_APP);
```
Have not found a problem with `undefined` but with `withOnyx` there was a default value `false` for this parameter.